shell/plugins/dashboard-store/actions.js

```diff
@@ -384,7 +384,7 @@ export default {
     }
 
     // No need to request the resources if we have them already
-    if (!opt.force && getters['havePaginatedPage'](type, opt)) {
+    if (!opt.transient && !opt.force && getters['havePaginatedPage'](type, opt)) {
```

Comment:
How do we define transient? 

I didn't find existing documentation of the `FindPageOpt` type.

Reply:
Typing errors everywhere.

- I've ensured the object where this is set now is of the correct type (which immediately throws an error)
- Added property to correct ActionFindPageArgs interface (resolves error)
- Reference the correct type in comment for action find page action (`FindPageOpt` was an old name for the new `ActionFindPageArgs`)
- Reference the correct type in comment for action find all (this was referenced find page one...)